WordPress/src/main/res/values-night/styles.xml

```diff
@@ -124,4 +124,58 @@
         <item name="android:background">?android:attr/listDivider</item>
     </style>
 
+    <!-- Zendesk Styles Override for Dark Theme -->
+
+    <style name="ZendeskActivityDefaultTheme" parent="WordPress.NoActionBar">
+        <item name="colorPrimary">#282828</item>
+        <item name="snackbarButtonStyle">@style/ZendeskSdkTheme.Snackbar.Button</item>
+        <item name="snackbarStyle">@style/Widget.MaterialComponents.Snackbar</item>
+    </style>
+
+    <style name="ZendeskSdkTheme.Light.RequestList.AvatarText" parent="@style/TextAppearance.MaterialComponents.Body1">
+        <item name="android:textSize">14sp</item>
+        <item name="android:lines">1</item>
+    </style>
+
+    <style name="ZendeskSdkTheme.Light.Request.Message.SystemMessage" tools:ignore="UnusedResources" parent="@style/TextAppearance.MaterialComponents.Body1">
+        <item name="android:lineSpacingExtra">8dp</item>
+        <item name="android:fontFamily">sans-serif</item>
+        <item name="android:textStyle">normal</item>
+    </style>
+
+    <style name="ZendeskSdkTheme.Light.RequestList.Subject" parent="@style/TextAppearance.MaterialComponents.Body1">
+        <item name="android:textSize">14sp</item>
+        <item name="android:lines">1</item>
+    </style>
+
+    <style name="ZendeskSdkTheme.Light.RequestList.Title" parent="@style/TextAppearance.MaterialComponents.Body1">
+        <item name="android:textSize">16sp</item>
+        <item name="android:lines">1</item>
+    </style>
+
+    <style name="ZendeskSdkTheme.Light.Request.Date" parent="@style/TextAppearance.MaterialComponents.Body1">
+        <item name="android:textColor">?attr/wpColorOnSurfaceMedium</item>
+        <item name="android:textSize">14sp</item>
+        <item name="android:fontFamily">sans-serif</item>
+        <item name="android:textStyle">normal</item>
+    </style>
+
+    <style name="ZendeskRequestListButton" parent="@style/WordPress.Button.Primary" />
```

Comment:
❓ I'm curious why these styles are only overridden in the `values-night` resource file and do not exist in the non-night version. None of these styles are specific to night-mode so if they are moved to the regular `/values/styles.xml` resource file you'd have a more consistent experience between light and dark mode. A good example of this is the differences in the formatting of the system message in your screenshots between light and dark themes:

![Screen Shot 2020-06-26 at 1 45 49 PM](https://user-images.githubusercontent.com/5810477/85895198-6df4a500-b7b3-11ea-91fb-afb8e756e5f6.png)

Moving all the styles that do not specifically override something special for dark mode to the main styles resource file will fix these inconsistencies, for example with the system message referenced above:

Light | dark 
-- | --
![Screenshot_1593200822](https://user-images.githubusercontent.com/5810477/85895270-9c728000-b7b3-11ea-9478-54fccd827544.png)|![Screenshot_1593200827](https://user-images.githubusercontent.com/5810477/85895275-9e3c4380-b7b3-11ea-978c-f7d7dbdd7939.png)